server.js

```diff
@@ -13,6 +13,8 @@ import { info, error } from './lib/loggers';
 import auth from './lib/auth';
 import graphqlErrorHandler from './lib/graphql-error-handler';
 
+import 'source-map-support/register';
```

Comment:
FWIW I noticed we're dropping these global registers in the [main index.js file](https://github.com/artsy/metaphysics/blob/master/index.js#L3).